library-docker/bin/restart-kiwix-serve.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+export PATH=/usr/local/bin:/usr/sbin:$PATH
+
+start () {
+  stop  
+  echo "Starting kiwix-serve..."
+  kiwix-serve --daemon --port=8000 --library --threads=16 --verbose --nodatealias library.kiwix.org.xml
```

Comment:
At this point this is an assumption that the cache in Varnish is in sync with kiwix-serve. Each time kiwix-serve is restarted, the Varnish cache should be cleaned.

Reply:
Should it? We already refresh the cache on the daily library operation that kills kiwix-serve.
If we refresh it on each restart, we'll loose some of the benefits of this for no apparent gain. Or is it likely that the library will be modified outside this daily task?